src/main/webapp/app/entities/result/result-detail.component.ts

```diff
@@ -40,4 +40,8 @@ export class ResultDetailComponent implements OnInit {
         });
         this.isLoading = false;
     }
+
+    filterFeedback(feedback: Feedback) {
```

Comment:
Is this still used? Seems the pipe is taking care of this task.

Reply:
Whoopsie, forgot to remove this one - was another approach in using pipes.